app/models/ability/administrator.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+# Privileges granted users with the administrator role
+class Ability::Administrator
+  include CanCan::Ability
+
+  prepend Ability::Shared::SuperUser
+  prepend Ability::Shared::PowerUser
+  prepend Ability::Shared::TagCreationUser
+  prepend Ability::Shared::SampleManagementUser
+
```

Comment:
Lets us group together associated permissions. Some of this is thinking ahead. (Ie. Super user privileges probably shouldn't be granted to people outside of PSD, but at the moment are as administrator is the higest level we have)